spec/Candid.md

```diff
@@ -1318,17 +1352,17 @@ Deserialisation at an expected type sequence `(<t'>,*)` proceeds by
 
  * checking for the magic number `DIDL`
  * using the inverse of the `T` function to decode the type definitions `(<t>,*)`
- * check that `(<t>,*) <: (<t'>,*)`, else fail
```

Comment:
Why is the subtyping check dropped here?

Reply:
This was an out-dated algorithm. We only check subtyping for reference types, which is covered by the `~>` derivation below. This is needed to support extensible variant.